tests/helpers.py

```diff
@@ -88,3 +88,50 @@ def diameter_over_neutral(self):
     @property
     def neutral_strand_count(self):
         return self._neutral_strand_count
+
+
+class MultiLineModel:
+    def __init__(self, conductors, is_secondary=False):
+        self._resistance = {}
```

Comment:
why not just use `self.resistance` etc?. we don't really need an internal attribute and accessor method unless we need to do some work to calculate the answer.

Reply:
yeah I did it this way to be consistent with `LineModel` and `ConcentricLineModel` classes. I can refactor them all to drop the internal attributes if wanted.